package Protos;


message ClassName {
	required string class_name = 1;
	required bool has_proto = 2;
	required uint32 size = 3;
}

message EmptyProto {

}

message TStringToUintPair {
    required string first = 1;
    required uint32 second = 2;
}

message TUintToUintPair {
    required uint32 first = 1;
    required uint32 second = 2;
}
